server/service/src/sync/translations/asset_catalogue_item.rs

```diff
@@ -0,0 +1,108 @@
+use repository::{
+    asset_catalogue_item_row::{AssetCatalogueItemRow, AssetCatalogueItemRowRepository},
+    ChangelogRow, ChangelogTableName, StorageConnection, SyncBufferRow,
+};
+
+use crate::sync::translations::asset_category::AssetCategoryTranslation;
+use crate::sync::translations::asset_class::AssetClassTranslation;
+use crate::sync::translations::asset_type::AssetTypeTranslation;
+
+use super::{
+    PullTranslateResult, PushTranslateResult, SyncTranslation, ToSyncRecordTranslationType,
+};
+
+// Needs to be added to all_translators()
+#[deny(dead_code)]
+pub(crate) fn boxed() -> Box<dyn SyncTranslation> {
+    Box::new(AssetCatalogueItemTranslation)
+}
+
+struct AssetCatalogueItemTranslation;
+
+impl SyncTranslation for AssetCatalogueItemTranslation {
+    fn table_name(&self) -> &'static str {
+        "asset_catalogue_item"
+    }
+
+    fn pull_dependencies(&self) -> Vec<&'static str> {
+        vec![
+            AssetClassTranslation.table_name(),
+            AssetCategoryTranslation.table_name(),
+            AssetTypeTranslation.table_name(),
+        ]
+    }
+
+    fn try_translate_from_upsert_sync_record(
+        &self,
+        _: &StorageConnection,
+        sync_record: &SyncBufferRow,
+    ) -> Result<PullTranslateResult, anyhow::Error> {
+        Ok(PullTranslateResult::upsert(serde_json::from_str::<
+            AssetCatalogueItemRow,
+        >(&sync_record.data)?))
+    }
+
+    fn change_log_type(&self) -> Option<ChangelogTableName> {
+        Some(ChangelogTableName::AssetCatalogueItem)
+    }
+
+    // Only translating and pushing on central server
```

Comment:
I think this comment is now wrong, Only translate when pulling from omSupply Central ?